wiki/Skinning/osu!mania/en.md

```diff
@@ -263,135 +292,164 @@ The following table is for all lengths (tail part).
-| Animatable   | Beatmap Skinnable |
-|:------------:|:-----------------:|
-| ![No][false] | ![No][false]      |
+![](img/mania-stage-right.png)
 
-**Notes:**
+| Versions | Animatable | Beatmap Skinnable | Blend Mode | Origin | Suggested SD Size |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| All | ![No][false] | ![No][false] | Normal | BottomRight | Max height: 768px |
+
+Notes:
 
 - This element is shown on the right side of the stage(s).
 - This is stretched to fit the stage height (allows for smaller images).
 
 ---
 
-_`mania-stage-bottom.png`_ (transparent; not included)
+`mania-stage-bottom.png`
 
-| Animatable   | Beatmap Skinnable |
-|:------------:|:-----------------:|
-| ![Yes][true] | ![No][false]      |
+| Versions | Animatable | Beatmap Skinnable | Blend Mode | Origin | Suggested SD Size |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| All | ![Yes][true] | ![No][false] | Normal | Bottom | - |
 
-**Notes:**
+Notes:
 
-- Animation name: `mania-stage-bottom-#.png`, replace # with an indexed count (start count at zero).
+- This is 0.625x smaller than the stage width.
+- Animation name: `mania-stage-bottom-{n}.png`.
 - Shown on the bottom (or top, if the stage is upside down) of the stage(s)
-- This element will **not** stretched to fit the stage width!
-- This is 0.625x smaller than the columns.
+- This element will not stretched to fit the stage width!
 - This element should be skinned for a 480px playfield height.
+- This element overlays the entire stage, including the notes.
 
 ---
 
-![](img/mania-stage-light.png "mania-stage-light.png")
+`mania-stage-light.png`
+
+![](img/mania-stage-light.png)
 
-| Animatable   | Beatmap Skinnable |
-|:------------:|:-----------------:|
-| ![Yes][true] | ![No][false]      |
+| Versions | Animatable | Beatmap Skinnable | Blend Mode | Origin | Suggested SD Size |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| All | ![Yes][true] | ![No][false] | Multiplicative | Bottom | Max height: 768px |
 
-**Notes:**
+Notes:
 
-- Animation name: `mania-stage-light-#.png`, replace # with an indexed count (start count at zero).
+- Animation name: `mania-stage-light-{n}.png`.
 - This is the lighting for the columns when the key is pressed.
 - This element is placed underneath the notes.
+- Tinted white by default.
+  - Use `ColourLight` to change this.
+- Positioning is set by the [skin.ini](/wiki/skin.ini)
+  - Use `LightPosition`.
 
 ---
 
-![](img/mania-stage-hint.png "mania-stage-hint.png")
+`mania-stage-hint.png`
+
+![](img/mania-stage-hint.png)
 
-| Animatable   | Beatmap Skinnable |
-|:------------:|:-----------------:|
-| ![No][false] | ![No][false]      |
+| Versions | Animatable | Beatmap Skinnable | Blend Mode | Origin | Suggested SD Size |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| All | ![Yes][true] | ![No][false] | Normal | Centre | - |
 
-**Notes:**
+Notes:
 
-- This is the graphical representation of the judgement line (or the hit area).
+- This is the graphical representation of the judgement line.
   - The judgement line is drawn in the middle of the image.
```

Comment:
Change `middle` to `centre`
Add
>- This is stretched to fit the stage width (allows for smaller images)

This is similiar to the mania-stage-left.png/-right.png case.